frontend/src/components/common/Button/ButtonLocation.tsx

```diff
@@ -0,0 +1,72 @@
+import { theme } from 'src/styles/theme';
+import styled from 'styled-components';
+import Icon from '../Icon/Icon';
+
+interface Props {
+  title?: string;
+  status: TStatus;
+  className?: string;
+  onClick: () => void;
+}
+
+type TStatus = 'add' | 'active' | 'inactive';
+type TStatusDetail = 'background-color' | 'color';
+
+const statusList: Record<TStatus, Record<TStatusDetail, string>> = {
+  add: {
+    'background-color': theme.color.white,
+    color: theme.color.primary200,
+  },
+  active: {
+    'background-color': theme.color.primary200,
+    color: theme.color.white,
+  },
+  inactive: {
+    'background-color': theme.color.white,
+    color: theme.color.primary200,
+  },
+};
+
+function ButtonLocation({ title, className, status, onClick }: Props) {
+  const isAdd = (): boolean => status === 'add';
```

Comment:
이 부분을 상수가 아닌 함수로 하신 이유가 있으신가요??

```suggestion
  const isAdd = status === 'add';
```

Reply:
이런 방법이 있었네요~ 항상 함수로만 작성하다보니 몰랐네요